packages/store/src/patterns/encodeKey.js

```diff
@@ -62,7 +74,11 @@ const dbEntryKeyToNumber = k => {
     bits ^= 0x8000000000000000n;
   }
   asBits[0] = bits;
-  return asNumber[0];
+  const result = asNumber[0];
+  if (is(result, -0)) {
+    return 0;
+  }
+  return result;
```

Comment:
Under what circumstances would a negative zero end up as a key in the store? Or is it just defense in depth?

Reply:
Just defense in depth.